demo/chbench/kafkasnap/README.md

```diff
@@ -0,0 +1,3 @@
+# kafkasnap service
```

Comment:
organizationally I would prefer if this was in a directory in `misc` with its own mzbuild.yml since it doesn't seem to actually be specifically related to chbench. I would like us to move away from using raw `build:` in mzcompose in general, it makes mzcompose hard to reproduce and slow.

Reply:
Can we move the Python code out of `misc`? Organizationally I find that confusing.

Happy to move this into a service under misc, although I've also been thinking of moving the `demo/chbench` stuff into `benchmarks/chbench`, since that seems more applicable.